### DISCUSSION TITLE
v0.10.4 Stable Release

### DISCUSSION THREAD

**Discussion by Amazinite** [endless-sky/endless-sky#9493] [Announcements]
This is a stable release, focusing on bug fixes and other small improvements.

These changes include:
* Various fixes and improvements to the new Coalition campaign intros.
* Outfits can now be given the "unique" attribute. The player will be warned when any such outfits are about to be lost.
* All controls and settings now have tooltips.
* It is now possible to recall the help messages for the currently active panel on demand.

There are also many other changes. For a full list, see the [changelog](https://github.com/endless-sky/endless-sky/blob/v0.10.4/changelog).
A special thanks to everyone who contributed to this release: @Amazinite, @Anarchist2, @Arachi-Lover, @bene-dictator, @hexagonrecursion, @Hurleveur, @LazerLit, @quyykk, @RisingLeaf, @Saugia, @tibetiroka, @TomGoodIdea, and @warp-core.

![image](https://github.com/endless-sky/endless-sky/assets/17688683/8d4be2f0-4332-4d22-bc5f-6221eebf4bef)

<hr /><em>This discussion was created from the release <a href='https://github.com/endless-sky/endless-sky/releases/tag/v0.10.4'>v0.10.4 Stable Release</a>.</em>

**Comment by SultanRTG** [comment]
The Coalition storyline is available to play?

**Reply by lumbar527** [reply]
Yes
